### PR TITLE
change workflow from release* to main

### PR DESCRIPTION
**Pull request by TimWhiteSource**

### CODE REVIEW

**Comment by TimWhiteSource** [comment]
approved

File: .github/workflows/github-action.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Whitesource Prioritize Java with Maven
 
 on:
   push:
-    branches: [ release* ]
+    branches: [ main ]
```
Comment (TimFranklin) [inline .github/workflows/github-action.yml:6]: changing to main is fine

**Review by TimFranklin — APPROVED** [review]

File: .github/workflows/github-action.yml (hunk 1 above)
Comment (TimFranklin) [reply .github/workflows/github-action.yml:6]: Swapping to main is fine

**Review by TimFranklin — APPROVED** [review]
swapping to main is approved

**Review by TimFranklin — APPROVED** [review]

**Comment by TimFranklin** [comment]
Swapping to main is fine

**Review by TimFranklin — APPROVED** [review]
a

**Review by TimFranklin — APPROVED** [review]
a

File: .github/workflows/github-action.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: Whitesource Prioritize Java with Maven
 
 on:
```
Comment (TimFranklin) [inline .github/workflows/github-action.yml:4]: Main

**Review by TimFranklin — APPROVED** [review]
approve

**Review by TimFranklin — APPROVED** [review]
yes

**Review by TimFranklin — APPROVED** [review]
a

**Review by TimFranklin — APPROVED** [review]
swapping to main is fine

**Pull request closed without merging**